warehouse/oidc/models/_core.py

```diff
@@ -39,6 +39,20 @@ def wrapper(ground_truth, signed_claim, all_signed_claims):
     return wrapper
 
 
+def _check_claim_invariant(value: Any):
+    """
+    Wraps a fixed value comparison into a three-argument function.
+
+    This is used solely to make claim verification compatible with "invariant"
+    comparison checks, like "claim x is always the literal `true` value".
+    """
+
+    def wrapper(ground_truth, signed_claim, all_signed_claims):
+        return value == signed_claim
```

Comment:
This is a little confusing because we need to define the `@property` for `ground_truth` to be present as an argument here, but then we don't actually use it when verifying the invariant.

I think this would make more sense if this was:

```suggestion
    def wrapper(ground_truth, signed_claim, all_signed_claims):
        return ground_truth == signed_claim == value
```

Reply:
Yeah, thanks for catching that! I agree completely.